src/content/reducers/types.js

```diff
@@ -1,14 +1,23 @@
+// @flow
```

Comment:
Most changes here are moving things around. Only meaningful change is forbidding a null Profile.